### PR TITLE
Add subscriptionField, initial pass at #21

### PR DESCRIPTION
**Pull request by tgriesser**
Interested to get feedback on the implementation here, it works slightly differently than the other types  since subscription has different semantics:

TODO:
- [ ] Add subscription example to kitchen sink

### CODE REVIEW

File: src/definitions/definitionBlocks.ts [hunk 1]
```diff
@@ -76,6 +83,7 @@ export interface NexusOutputFieldConfig<
 
 export type NexusOutputFieldDef = NexusOutputFieldConfig<string, any> & {
   name: string;
+  subscribe?: GraphQLFieldResolver<any, any>;
```
Comment (Devlin556) [inline src/definitions/definitionBlocks.ts]: Why the `subscribe` field should have `GraphQLFieldResolver` type here?

**Comment by nevescode** [comment]
I'm really looking forward to it! 👍

**Comment by Knaackee** [comment]
When did subscription support will get released? I really need this.
Thanks for your great work!

**Comment by tgriesser** [comment]
Released this along with a few other changes in 0.11

I'm calling the `mutationField` feature alpha. It should work but it's undocumented for now. Kick the tires on it and let me know what you think!

**Pull request merged**